src/AddressRegistry.Api.Backoffice/AddressRegistry.Api.Backoffice.csproj

```diff
@@ -0,0 +1,50 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+  <Import Project="..\..\packages\Be.Vlaanderen.Basisregisters.Build.Pipeline\Content\Be.Vlaanderen.Basisregisters.Build.Pipeline.Settings.App.props" />
+
+  <PropertyGroup>
+    <ServerGarbageCollection>true</ServerGarbageCollection>
+    <EnableDefaultContentItems>false</EnableDefaultContentItems>
+    <LangVersion>latest</LangVersion>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <DocumentationFile>bin\Debug\netcoreapp3.1\AddressRegistry.Backoffice.Legacy.xml</DocumentationFile>
+    <NoWarn>1701;1702;1705;1591</NoWarn>
+    <DefineConstants>TRACE;DEBUG;NETCOREAPP;NETCOREAPP3_1</DefineConstants>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
+    <DocumentationFile>bin\Release\netcoreapp3.1\AddressRegistry.Backoffice.Legacy.xml</DocumentationFile>
+    <NoWarn>1701;1702;1705;1591</NoWarn>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Content Include="appsettings.json" CopyToOutputDirectory="Always" />
+    <Content Include="appsettings.*.json" CopyToOutputDirectory="Always" />
+    <Content Include="paket.template" CopyToOutputDirectory="Always" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\AddressRegistry.Infrastructure\AddressRegistry.Infrastructure.csproj" />
+    <ProjectReference Include="..\AddressRegistry.Projections.Legacy\AddressRegistry.Projections.Legacy.csproj" />
+    <ProjectReference Include="..\AddressRegistry\AddressRegistry.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Compile Include="..\..\SolutionInfo.cs" Link="Properties\SolutionInfo.cs" />
+  </ItemGroup>
+
+  <!-- ToDo: remove and include as nuget packages -->
+  <ItemGroup>
+    <Reference Include="CrabEdit.Client">
+      <HintPath>..\..\..\poc-crab-edit\src\CrabEdit.Client\bin\Debug\netstandard2.0\CrabEdit.Client.dll</HintPath>
+    </Reference>
+    <Reference Include="CrabEdit.Infrastructure">
+      <HintPath>..\..\..\poc-crab-edit\src\CrabEdit.Client\bin\Debug\netstandard2.0\CrabEdit.Infrastructure.dll</HintPath>
+    </Reference>
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="RestSharp.Serializers.NewtonsoftJson" Version="106.11.4" />
```

Comment:
This should be in in paket.dependencies

Reply:
will be removed once CrabEdit is included as a package